packages/example/pages/_app.tsx

```diff
@@ -83,11 +83,16 @@ type AccentColor = typeof accentColors[number];
 const radiusScales = ['large', 'medium', 'small', 'none'] as const;
 type RadiusScale = typeof radiusScales[number];
 
+const transitionScales = ['large', 'small', 'fade', 'none'] as const;
```

Comment:
I think `fade` should be renamed to `small` because this API is likely to be used for transitions other than the modal. If so, should `small` be renamed to `medium`?